src/renderer/features/player/hooks/use-center-controls.ts

```diff
@@ -668,10 +754,8 @@ export const useCenterControls = (args: { playersRef: any }) => {
       });
 
       mpris.requestVolume((_e: any, data: { volume: number }) => {
-        const currentVolume = usePlayerStore.getState().volume;
-        const resultingVolume = data.volume + currentVolume;
+        let newVolume = data.volume * 100;
```

Comment:
newVolume should be rounded using `Math.round()` so we don't get values like 50.000000000.

Reply:
Also make sure to notify other mpris clients of the volume change by again calling `mpris?.updateVolume(data.volume);` when receiving the volume request, so they can keep track of Feishins volume.
This is important, so clients like playerctl work properly.